README.md

```diff
@@ -20,7 +20,7 @@ pip install -r requirements.txt
 
 ## Script Usage
 
-The main purpose of `plot_srt_stats.py` script is to visualize all the SRT core statistics produced during experiments by one of the [testing applications](https://github.com/Haivision/srt/blob/master/docs/stransmit.md) or third-party solutions supporting SRT protocol. Depending on whether this statistics is collected on sender or receiver side, the data plotted may vary.
+The main purpose of `plot_srt_stats.py` script is to visualize all the SRT core statistics produced during experiments by one of the [testing applications](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md) or third-party solutions supporting SRT protocol. Depending on whether this statistics is collected on sender or receiver side, the data plotted may vary.
```

Comment:
Suggested change 1: 
"by one of the [testing applications](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md) or third-party solutions" ->
"by one of the testing applications like [srt-xtransmit](https://github.com/maxsharabayko/srt-xtransmit) and [srt-live-transmit](https://github.com/Haivision/srt/blob/master/docs/apps/srt-live-transmit.md) or third-party solutions"

Suggested change 2:
"to visualize all the SRT core statistics" ->
"to visualize SRT core statistics"